coop_shift/views/view_res_partner.xml

```diff
@@ -20,6 +20,7 @@
     <record id="view_res_partner_shift_form" model="ir.ui.view">
         <field name="model">res.partner</field>
         <field name="inherit_id" ref="base.view_partner_form"/>
+        <field name="groups_id" eval="[(5, )]"/>
```

Comment:
It's better to do this in a migration script, for many reasons:

1) Clearer code
2) Ability to customize in db